server/views/pages/vary-approve/approved.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components//button/macro.njk" import govukButton %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set pageTitle = applicationName + " - confirm variation approved" %}
+{% set pageId = "confirm-variation-approved-page" %}
+{% set hideLicenceBanner = true %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{ govukPanel({
+                titleText: "The licence variation for " + licence.forename + " " + licence.surname + " has been approved."
+            }) }}
+
+            <h2 class="govuk-heading-m">What happens next</h2>
+            <p class="govuk-body" id="sent-to">
```

Comment:
I'll fix this. No Axe check yet as not called by int tests - so probably would fail. Waiting until finished journey is in place with diffs/accordion before doing an int test.